### PR TITLE
Gumlet data

### PR DESCRIPTION
**Pull request by mdhuprna**

### CODE REVIEW

File: push-athena-data-to-rds/push-athena-data-to-rds.go [hunk 1]
```diff
@@ -14,76 +15,76 @@ import (
...
 }
 
-func getVarnishDataFromAthena(athenaDBName string, athenaTableName string, s3Location string, queryParams map[string]string) {
-	completeS3Location := utils.CompleteS3Location(s3Location, "varnish", queryParams)
-
-	query, queryErrMsg := athena.VarnishDataQuery(queryParams, athenaDBName, athenaTableName)
-
-	if queryErrMsg.Err != nil {
-		fmt.Println(queryErrMsg.Message, queryErrMsg.Err)
+func getTableName(dataSource string) string {
```
Comment (gja) [inline push-athena-data-to-rds/push-athena-data-to-rds.go]: I see the same switchCase here nearly three times. At some point, you sould replace this with

function getConfig(datasource string) {
  return { table-name: xyz, database: xyz, query: Query}
}

File: push-athena-data-to-rds/push-athena-data-to-rds.go [hunk 2]
```diff
@@ -142,8 +143,10 @@ func checkVariablesPresence() bool {
 	_, isPrimaryDomainTable := os.LookupEnv("PRIMARY_DOMAIN_TABLE")
 	_, isVarnishTable := os.LookupEnv("VARNISH_TABLE")
 	_, isHaproxyTable := os.LookupEnv("HAPROXY_TABLE")
+	_, isGumletDB := os.LookupEnv("GUMLET_DB")
+	_, isGumletTable := os.LookupEnv("GUMLET_TABLE")
 
-	if isS3PathPresent && isBucketNamePresent && isBadgerHost && isBadgerAuth && isCloudflareDB && isAlbDB && isVarnishTable && isAssettypeTable && isPrimaryDomainTable && isHaproxyTable {
+	if isS3PathPresent && isBucketNamePresent && isBadgerHost && isBadgerAuth && isCloudflareDB && isAlbDB && isVarnishTable && isAssettypeTable && isPrimaryDomainTable && isHaproxyTable && isGumletDB && isGumletTable {
```
Comment (gja) [inline push-athena-data-to-rds/push-athena-data-to-rds.go]: what is this?

**Review by gja — DISMISSED** [review]

File: push-athena-data-to-rds/push-athena-data-to-rds.go (hunk 2 above)
Comment (mdhuprna) [reply push-athena-data-to-rds/push-athena-data-to-rds.go]: I am checking if all the required environment variable are present.

**Comment by gja** [comment]
Why not just do for i in [VAR1, VAR2], abort if VAR.missing

--
Tejas Dinkar
Head of Technology
Quintype

**Comment by gja** [comment]
Looks good.

--
Tejas Dinkar
Head of Technology
Quintype

**Comment by gja** [comment]
--
Tejas Dinkar
Head of Technology
Quintype

**Pull request merged**